### ISSUE TITLE
Creating a room map

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#15]
## User story
1. As a SD
2. I want to create a map for the robot using the SMET software
3. So that robot navigation is more convenient in a room where I work regularly.  

## Acceptance criteria
* SMET software should be able to navigate the robot

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by MeikeBloecher** [comment]
![Map_withLine](https://user-images.githubusercontent.com/104436432/168793787-755500d4-5354-4bd2-a050-59a2c6974c04.png)
I created a map of the thursday working room and drew a first course, the purple line. The dots are either chair or table legs. The picture shows the loaded map in the lidar loc software related to issue #22.
